SOF-7136: property manifest for Hubbard V nearest neighbors

pranabdas opened the pull request.

pranabdas committed:
SOF-7136: property manifest for Hubbard V nearest neighbors

pranabdas committed:
SOF-7136: include hubbard_v_nn property name

pranabdas committed:
SOF-7136: separate schemas for Hubbard V and V nearest neighbors for stricter validation

timurbazhirov commented on schema/properties_directory/non-scalar/hubbard_v_nn.json:
```diff
@@ -0,0 +1,35 @@
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Hubbard V parameters for nearest neighbors",
+    "description": "Hubbard V value in eV for nearest neighbors used in hp.x output parsing",
+    "properties": {
```
Maybe let's just reuse allOf:

```json
                "allOf": [
                    {
                        "$ref": "hubbard_v.json"
                    }
```

And keep description

timurbazhirov commented on schema/properties_directory/non-scalar/hubbard_v.json:
```diff
@@ -24,6 +24,7 @@
                     "id2",
                     "atomicSpecies",
                     "atomicSpecies2",
+                    "distance",
```
Let's have v and v_nn to be identical from the data structure standpoint, the latter is just for convenience purposes.

We could do one of the following:

- get the default highest occupied orbital for each element per https://github.com/Exabyte-io/periodic-table.js/blob/dev/periodic-table.json and insert during parsing
- try to read this information from XML file when dealing with QE

pranabdas committed:
SOF-7136: reuse hubbard_v as hubbard_v_nn

keep the datastructures same, remove properties in difference set from being required
